projects/components/src/splitter/splitter.component.scss

```diff
@@ -0,0 +1,81 @@
+@import 'mixins';
+
+:host {
+  width: 100%;
+  height: 100%;
+}
+
+.splitter-container {
+  display: flex;
+  width: 100%;
+  height: 100%;
+
+  &.horizontal {
+    flex-direction: row;
+  }
+
+  &.vertical {
+    flex-direction: column;
+  }
+
+  .splitter-content {
+    display: flex;
+  }
+
+  .splitter-content > p {
+    width: 100%;
+    height: 100%;
```

Comment:
nit: can be replaced with `fill-container` and all the other places as well

Reply:
done